library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayPrivacy.kt

```diff
@@ -129,8 +133,14 @@ enum class SessionReplayPrivacy {
             MapperTypeWrapper(Button::class.java, buttonMapper.toGenericMapper()),
             MapperTypeWrapper(EditText::class.java, editTextViewMapper.toGenericMapper()),
             MapperTypeWrapper(TextView::class.java, textMapper.toGenericMapper()),
-            MapperTypeWrapper(ImageView::class.java, imageMapper.toGenericMapper())
+            MapperTypeWrapper(ImageView::class.java, imageMapper.toGenericMapper()),
+            MapperTypeWrapper(AppCompatToolbar::class.java, unsupportedViewMapper.toGenericMapper())
```

Comment:
let's try to do this more generic like we discussed. Maybe to be applied for all the implementations of the `View` type that do not have an explicit mapper yet.